microovn/ovn/join.go

```diff
@@ -98,33 +99,9 @@ func Join(ctx context.Context, s state.State, initConfig map[string]string) erro
 
 	// Enable OVN central (if needed).
 	if srvCentral < 3 {
```

Comment:
Not a problem concerning this PR, but this is actually a bug. The system should probably create the cluster when there is 1 node, and hold off until there are 3 before joining two more central services to the cluster, rather than knowingly creating a cluster incapable of handling partitions when at 2 nodes.

I filed https://bugs.launchpad.net/microovn/+bug/2076436 for this.